src/merkle/ihr.rs

```diff
@@ -230,9 +230,9 @@ impl FirstPassIhr {
 impl Ihr {
     /// Do the second pass of the IHR computation. This must be called on the result
     /// of first pass.
-    pub fn compute_pass2(first_pass: FirstPassIhr, ty: &FinalArrow) -> Ihr {
+    pub fn compute_pass2(imr: Imr, ty: &FinalArrow) -> Ihr {
```

Comment:
Without a "pass 1" this could potentially be renamed `compute_ihr`.

Reply:
Ah, I like that.

Reply:
Changed it to `from_imr`.